tests/conftest.py

```diff
@@ -107,11 +107,18 @@ def git_clone_fixtures(url: str, commit_hash: str, location: str) -> None:
 
 def pytest_sessionstart(session: Session) -> None:
 
-    fixtures_location = "tests"
+    fixtures_location = "tests/fixtures"
+
+    if not os.path.exists(fixtures_location):
+        os.mkdir(fixtures_location)
```

Comment:
Better to just try and make the directory than to check if it exists (avoids [TOCTOU](https://en.wikipedia.org/wiki/Time-of-check_to_time-of-use) attacks, not that it's relevant here):

```suggestion
    try:
        os.mkdir(fixtures_location)
    except FileExistsError:
        pass
```

Reply:
Thanks for this context. I will update.